packages/web-components/src/components/va-process-list/va-process-list.tsx

```diff
@@ -19,7 +19,7 @@ export class VaProcessList {
   /**
    * Whether or not the component will use USWDS v3 styling.
    */
-  @Prop() uswds?: boolean = false
+  @Prop() uswds?: boolean = true
```

Comment:
We may need an exception here. v3 `va-process-list` expects `va-process-list-item` as its children while v1 expects `li`. This will break v1s being auto-upgraded to v3

Reply:
Good call out here @ataker. I think what we'll do is go with @jamigibbs's suggestion to set all of the necessary instances in vets-website to `uswds=false`. There aren't that many, and it unblocks our release plans for next week.